From: Chris Pick <chris@chrispick.com>
Date: Mon, 11 May 2015 18:30:14 -0400
Subject: Don't have Makefile initialize and update PolarSSL submodule

When building the Debian package, the orig source tarball already contains the
proper PolarSSL version.  It doesn't however contain the git data and so can't
execute any git commands.
---
 Makefile | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/Makefile b/Makefile
index f9aecef..7a10b65 100644
--- a/Makefile
+++ b/Makefile
@@ -16,19 +16,6 @@ TEST_SRC=$(wildcard tests/*_tests.c)
 TESTS=$(patsubst %.c,%,${TEST_SRC})
 MAKEOPTS=OPTFLAGS="${NOEXTCFLAGS} ${OPTFLAGS}" OPTLIBS="${OPTLIBS}" LIBS="${LIBS}" DESTDIR="${DESTDIR}" PREFIX="${PREFIX}"
 
-# Prepare PolarSSL git submodule
-# 
-# - Perform src/polarssl submodule init and update, if necessary.  This is executed
-#   upon every make invocation, and must be done before the SOURCES variable, above
-#   is lazily evaluated, or none of the src/polarssl source files will be found
-
-ifdef $($(shell									\
-	if git submodule status | grep '^-'; then				\
-	    echo "PolarSSL; init and update git submodule" 1>&2;		\
-	    git submodule init && git submodule update;				\
-	fi ))
-endif
-
 all: bin/mongrel2 tests m2sh procer
 
 dev: CFLAGS=-g -Wall -Isrc -Wall -Wextra $(OPTFLAGS) -D_FILE_OFFSET_BITS=64
